bundles/org.openhab.binding.bluetooth.bluez/src/main/java/org/openhab/binding/bluetooth/bluez/handler/DBusBlueZAdapterConfiguration.java

```diff
@@ -12,14 +12,15 @@
  */
 package org.openhab.binding.bluetooth.bluez.handler;
 
-import org.openhab.binding.bluetooth.BaseBluetoothBridgeHandlerConfiguration;
-
 /**
- * Configuration properties class.
+ * Configuration properties for a bridge.
  *
- * @author Hilbrand Bouwkamp - Initial contribution
+ * @author Benjamin Lafois - Initial contribution
  */
-public class BlueZAdapterConfiguration extends BaseBluetoothBridgeHandlerConfiguration {
+public class DBusBlueZAdapterConfiguration {
```

Comment:
please leave this file unchanged and adjust your code to use the inherited `backgroundDiscovery` config instead of the `discovery` config you define here.